camel.go

```diff
@@ -83,3 +89,17 @@ func isLower(c byte) bool {
 func isDigit(c byte) bool {
 	return c >= '0' && c <= '9'
 }
+
+func toLower(c byte) byte {
+	if isUpper(c) {
+		return c + ('a' - 'A')
```

Comment:
I'm 99.99% sure the compiler sees `'a' - 'A'` as a constant expression, because it can be expressed as `const a = 'a' - 'A'`.